go.mod

```diff
@@ -5,6 +5,7 @@ go 1.20
 require (
 	github.com/conductorone/baton-sdk v0.1.14
 	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
+	github.com/spf13/cobra v1.8.0
```

Comment:
Why did this change?

Reply:
After running go mod tidy / go mod vendor I found those changes